docs/can-guides/commitment/recipes/canvas-clock/canvas-clock.md

```diff
@@ -254,9 +253,9 @@ In this section, we will:
 Update the __JavaScript__ tab to:
 
 @sourceref ./4-second-hand.js
-@highlight 1-3,22-48,only
+@highlight 3-5,31-57,only
```

Comment:
I think this should be `@highlight 3-5,31-58,only`

Reply:
Done!